src/compile/selection/selection.ts

```diff
@@ -90,10 +90,10 @@ export function parseUnitSelection(model: UnitModel, selDefs: Dict<SelectionDef>
     }
 
     name = varName(name);
-    const selCmpt = selCmpts[name] = extend({}, selDef, {
+    const selCmpt = selCmpts[name] = {...selDef,
```

Comment:
I would add a line break between `{` and `...`

Reply:
Done.